src/main/java/mods/eln/Eln.java

```diff
@@ -297,11 +296,16 @@
 
 	double stdHalfLife = 2 * Utils.minecraftDay;
 
+	public static boolean wailaEasyMode = false;
+
 	@EventHandler
 	public void preInit(FMLPreInitializationEvent event) {
 		
-		achNetwork = NetworkRegistry.INSTANCE.newSimpleChannel("achChannel");
-		achNetwork.registerMessage(AchievePacketHandler.class, AchievePacket.class, 0, Side.SERVER);
+		elnNetwork = NetworkRegistry.INSTANCE.newSimpleChannel("eln");
```

Comment:
As mentioned below, as long as someone doesn't try using a different version than the server has it should be fine.